google-cloud-spanner/src/main/java/com/google/cloud/spanner/SessionPool.java

```diff
@@ -2162,9 +2582,15 @@ enum Position {
   @GuardedBy("lock")
   private long numSessionsAcquired = 0;
 
+  @GuardedBy("lock")
+  private long numMultiplexedSessionsAcquired = 0;
```

Comment:
This does not appear to be updated anywhere in this PR. Is that intentional? (I can imagine it is, as it does not include the code for acquiring the session, but checking just to be sure)

Reply:
Yeah, I added it since I was also adding `numMultiplexedSessionsReleased`. the code that uses this constant will be in a separate PR.

Reply:
ack